tools/cbmc/proofs/ARP/ARPGetCacheEntry/ARPGetCacheEntry_harness.c

```diff
@@ -0,0 +1,17 @@
+/* FreeRTOS includes. */
+#include "FreeRTOS.h"
+#include "queue.h"
+
+/* FreeRTOS+TCP includes. */
+#include "FreeRTOS_IP.h"
+#include "FreeRTOS_IP_Private.h"
+#include "FreeRTOS_ARP.h"
+
+
+void harness()
+{
+  uint32_t ulIPAddress;
+  MACAddress_t xMACAddress;
+
+  eARPLookupResult_t xRes = eARPGetCacheEntry( &ulIPAddress, &xMACAddress );
```

Comment:
Probably no point capturing the return value if you are not doing anything with it.  But is there an interesting check we could do here?  I know that is a question about functional correctness and not memory safety, and is beyond the initial scope of your work.

The same question occurs to me about other test harnesses that capture return values.

Reply:
I will drop the return value. Functional correctness testing, from my point of view, would require a lot more work here to setup the Cache used inside in a more meaningful way than --nondet-static does.

Reply:
Let's focus on memory safety.  Functional correctness can come later.